doc/restcli.md

```diff
@@ -0,0 +1,15 @@
+# CLI REST endpoint
+
+Livepeer node exposes REST endpoint which is used by `livepeer_cli` for communication with node.
```

Comment:
It's not a REST endpoint, we can just say "HTTP endpoint" - same goes for the doc name, can rename to `httpcli.md`

Reply:
fixed